Source/FaerieInventoryContent/Private/Extensions/InventorySpatialGridExtension.cpp

```diff
@@ -257,26 +257,38 @@ bool UInventorySpatialGridExtension::RotateItem(const FInventoryKey& Key)
 	if (ItemShape.IsSymmetrical()) return false;
 
 	FFaerieGridPlacement NewPlacement = GetStackPlacementData(Key);
+	auto PrevRot = NewPlacement.Rotation;
 	NewPlacement.Rotation = GetNextRotation(NewPlacement.Rotation);
-
-	const FFaerieGridShape NewShape = ApplyPlacement(ItemShape, NewPlacement);
-
+	if(NewPlacement.Rotation == ESpatialItemRotation::None && PrevRot == ESpatialItemRotation::Two_Seventy)
```

Comment:
Ignore this for now, because the none rotation does nothing, when we apply the placement from the old origin nothing gets transformed so we wouldnt have a valid min point to choose as the new origin, have to figure out how to get it to properly revert back to original origin when going back to none without affecting shapes that are starting from none